config.json

```diff
@@ -2357,6 +2357,28 @@
           "macros"
         ],
         "difficulty": 8
+      },
+      {
+        "slug": "pov",
+        "name": "POV",
+        "uuid": "ABC93451-451D-4497-B590-68CBF82E0A20",
```

Comment:
I'm not sure if this matters on Exercism, but since all the other uuids are downcase, let's downcase this one too:
```suggestion
        "uuid": "abc93451-451d-4497-b590-68cbf82e0a20",
```

Reply:
Oh, I see we got an answer from the config file linter 😬 
<img width="1033" alt="Screen Shot 2021-06-07 at 21 22 40" src="https://user-images.githubusercontent.com/7852553/121076284-826c5a80-c7d6-11eb-9add-1541d97932e0.png">